src/components/Button/Button.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import className from 'classnames';
+import './Button.css';
+
+/**
+ * A Simple rounded button
+ */
+const Button = ({ size, color, outline, disabled, children }) => {
```

Comment:
need to use either `...rest` or `className` so that we can use utility classes on components as well. A use case would be adding margin to the button when it's used in context

Reply:
so maybe add a `className` prop and add it to the `className()` resolver? I suppose that makes it extensible for simple components, but I'm not sure how it would work for complex ones with many different tags.